src/sentry/static/sentry/app/views/settings/components/forms/fieldFromConfig.jsx

```diff
@@ -86,7 +88,7 @@ export default class FieldFromConfig extends React.Component {
 
         return <TextField {...props} />;
       case 'number':
-        return <InputField {...props} type="number" />;
+        return <NumberField {...props} />;
```

Comment:
Should we also add the `DateTimeField` and `RadioBoolean ` fields into this?

Reply:
I'm going to leave them out until the need arises.